docs/pages/assetFiles.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Asset Files
+nav_order: 4
+---
+Asset files, by default, will load from the `dymaptic.GeoBlazor.Core` library at `./_content/dymaptic.GeoBlazor.Core/assets`.
+
+If for some reason you want to bundle your own assets, you would change this by adding an environment variable or setting
+in `appsettings.json` of `"ArcGISAssetsPath": "...your/web/assets/url"`
```

Comment:
Some more text for you:

Note that this `ArcGISAssetsPath` can also be a full URL if your assets are hosted on your Enterprise portal, for example. Most people won't need to change this, but here are the [docs](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath) for this property.